pkg/githubclient/types.go

```diff
@@ -53,17 +52,6 @@ func newRepositoryFile(ghContent *github.RepositoryContent) (RepositoryFile, err
 		return RepositoryFile{}, microerror.Maskf(executionError, "expected non nil argument but got %#v", ghContent)
 	}
 
-	// Validate type.
-	{
-		typ, err := toString(ghContent.Type)
-		if err != nil {
-			return RepositoryFile{}, microerror.Mask(err)
-		}
-
-		if strings.ToLower(typ) != "file" {
-		}
-	}
-
```

Comment:
What was wrong with that?

Reply:
The linter complained that the one condition was empty. When I removed it `typ` was unused so I had to drop the whole block.

Reply:
Oh shit yeah. It should return an error. I'm happy we have all that stuff employed.

Reply:
Can you add it back with the error handling in case this is important? I am not sure what kind of error to return. Miss some context here. ❤️

Reply:
Sure. All good!